### DISCUSSION TITLE
disable create button

### DISCUSSION THREAD

**Discussion by anditsung** [laravel/nova-issues#4876] [Ideas & Feature Requests]
<img width="1047" alt="Screen Shot 2022-10-06 at 09 51 19" src="https://user-images.githubusercontent.com/1090413/194203269-c4aad3c4-8d18-4de0-b7ae-1e03be829d86.png">

is it possible to hide create resource button when the resource is access via resource?

phone model
```php
class Phone extends Model
{
    use HasFactory;

    public function types()
    {
        return $this->hasMany(PhoneType::class);
    }
}
```

phonetype model
```php
class PhoneType extends Model
{
    use HasFactory;

    public function phone()
    {
        return $this->belongsTo(Phone::class);
    }
}
```

after adding this to phonetype nova class i still can see the create button. 
```php
public static function authorizedToCreate(Request $request)
    {
        if ($request->viaResource) {
            return false;
        }
        return true;
    }
```

possible to create a logic to hide disable button base on parent resource?

**Comment by crynobone** [comment]
Hi there,

Add the moment you can do the following under `PhonePolicy`:

```php
public function addPhoneType(User $user, Phone $phone)
{
    return false;
}
```

**Comment by crynobone** [comment]
Using `$request->viaResource` is now possible on 4.16
